Dockerfile

```diff
@@ -24,6 +24,10 @@ FROM node:${NODE_VERSION}
 
 MAINTAINER FIWARE STH Team. Telefónica I+D
 
+RUN apt-get update \
```

Comment:
Needs a similar adaptation to the one done recently in other node related respositories.

Reply:
Not sure if this is the case here. The other repos have an option to download a zip from GitHub, STH-Comet does not. Ideally the Docker file can be updated and re-written to follow the same paradigm _at some point_, but this wasn't the scope or purpose of this PR. The retrieval/set-up should be a different layer IMHO.

Ensuring that Curl and/or Wget is available within the image is still useful however since it can be used for checking the heartbeat/health.

Note that things like PM2 are still hard coded in this image.

Reply:
In fat, looking to the original Dockerfile, curl and wget are not used at all.

Maybe this addition (which makes sense in the IOTAs) doesn't make sense for STH? Should the PR be closed?